src/chains/eos_1_8/templates/chainActions/createEscrow_reclaim.ts

```diff
@@ -33,17 +33,18 @@ export const decomposeAction = (action: EosActionStruct): EosDecomposeReturn =>
   if (name === actionName && data?.reclaimer && data?.dapp && data?.sym) {
     // If there's more than 1 authorization, we can't be sure which one is correct so we return null
     const auth = getFirstAuthorizationIfOnlyOneExists(authorization)
-    const returnData: createEscrowReclaimParams = {
+    const returnData: Partial<createEscrowReclaimParams> = {
       accountName: toEosEntityName(data.reclaimer),
       appName: data.dapp,
       contractName: toEosEntityName(account),
       permission: toEosEntityNameOrNull(auth?.permission),
```

Comment:
PLease make sure that you use the ...ifOnlyOne.. function where you need to. Seems like there are places misisng